test/zdtm.py

```diff
@@ -331,8 +331,7 @@ def decode_flav(i):
 
 def tail(path):
     p = subprocess.Popen(['tail', '-n1', path], stdout=subprocess.PIPE)
-    out = p.stdout.readline()
-    p.wait()
+    out, err = p.communicate()
```

Comment:
`err` is not used, we could use `unused` or `_` instead

Reply:
Fixed